### PR TITLE
Governance profile page and delegates

### PR DESCRIPTION
**Pull request by cjinghong**

### CODE REVIEW

File: governance/public/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Ribbon Finance NFT Drop is NFT Airdrop for Ribbon Finance"
```
Comment (kenchangh) [inline governance/public/index.html:10]: nit: change description

File: governance/public/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Ribbon Finance NFT Drop</title>
```
Comment (kenchangh) [inline governance/public/index.html:33]: title change

**Comment by kenchangh** [comment]
Playing around the app, I've got a few comments:
- The stutter [fixed in this commit](https://github.com/ribbon-finance/ribbon-frontend/commit/cad58f8a7b18c1a5cf9bcd6ae7b99ca2865581d2) is present in governance

File: governance/src/components/Profile/ProfileActivity.tsx [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    [account]
+  );
+
+  const renderActivityData = useCallback((activity: ActivityType) => {
+    switch (activity.type) {
+      case "stake":
+      case "unstake":
+        return (
+          <Title>
+            {formatAmount(parseFloat(formatUnits(activity.amount!, 18)))} RBN
```
Comment (kenchangh) [inline governance/src/components/Profile/ProfileActivity.tsx:197]: Could we refactor these function calls into `formatBigNumber` or something?

File: governance/src/components/Wallet/AccountStatus.tsx [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+    deactivateWeb3();
+    onCloseMenu();
+  }, [deactivateWeb3, onCloseMenu, connector]);
+
+  const renderButtonContent = () =>
+    active && account ? (
+      <>
+        <Indicator connected={active} />
+        <WalletButtonText connected={active}>
+          {truncateAddress(account)} <ButtonArrow isOpen={isMenuOpen} />
```
Comment (kenchangh) [inline governance/src/components/Wallet/AccountStatus.tsx:317]: Needs to include the changes that [displays ENS](https://github.com/ribbon-finance/ribbon-frontend/commit/766170d3c8211890cfc8201758f7af0eefbd4882)

File: governance/src/constants/constants.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+export const GovernanceChainID = isDevelopment()
+  ? CHAINID.ETH_KOVAN
+  : CHAINID.ETH_MAINNET;
+
+export const RibbonOGAddress = isDevelopment()
+  ? "0x81Eb8D19277da74BA7a1F804A79e1bC31CeF92C1"
+  : "0xf513bd4bA753548505A245be1877C39A4f0b5c96";
+
+export const getColorwayFromTokenID = (tokenId: number) => {
```
Comment (kenchangh) [inline governance/src/constants/constants.ts:11]: Do we still need to include this?

File: governance/src/components/Profile/ProfileActivity.tsx (hunk 3 above)
Comment (cjinghong) [reply governance/src/components/Profile/ProfileActivity.tsx:197]: Actually `formatBigNumber` and `formatAmount` are quite different in their utility in the respect where `formatAmount` turns the number into such as `138K` or `28M` whereas `formatBigNumber` would have to turn them into `138,398` and `28,284,283`  respectively. The reason I am using `formatUnits` here before passing into `formatAmount` is that `formatBigNumber` returns back number in a string that had been "tainted" with the dot `,`, and `formatAmount` require them in number format.

Comment (cjinghong) [reply governance/src/components/Profile/ProfileActivity.tsx:197]: But I guess we can have another utility that takes in BigNumber and return them in the `K` and `M` format

**Review by kenchangh — APPROVED** [review]

**Pull request merged**